doc/source/conf.py

```diff
@@ -258,6 +258,17 @@ def setup(app):
 
 # Additional stuff for the LaTeX preamble.
 latex_elements['preamble'] = r'''
+\newfontfamily\FontForChinese{FandolSong-Regular}[Extension=.otf]
+\catcode`琴\active\protected\def琴{{\FontForChinese\string琴}}
+\catcode`春\active\protected\def春{{\FontForChinese\string春}}
+\catcode`鈴\active\protected\def鈴{{\FontForChinese\string鈴}}
+\catcode`猫\active\protected\def猫{{\FontForChinese\string猫}}
+\catcode`傅\active\protected\def傅{{\FontForChinese\string傅}}
+\catcode`立\active\protected\def立{{\FontForChinese\string立}}
+\catcode`业\active\protected\def业{{\FontForChinese\string业}}
+\catcode`（\active\protected\def（{{\FontForChinese\string（}}
+\catcode`）\active\protected\def）{{\FontForChinese\string）}}
```

Comment:
Hmm, isn't there a way to blanket accept chinese characters?  I would suggest `textcomp` might help, but I am sure it is already used and this is something more complicated :).

Reply:
`textcomp` is for traditional TeX, i.e. `pdflatex`, (or `latex+dvipdfmx`) which works with TeX-specific font files each supporting 256 glyphs max (and even in this limited area, `textcomp` is much less efficient than what one would tend to think, for example it appeared recently that with many commonly used fonts in TeX, the straight backtick often used in code listings was picked up from LaTeX fallback Latin Modern font, despite it being available in the document monospace font; these things can go unnoticed for 20 years).

Here with `xelatex` and OpenType/TrueType fonts, yes indeed Chinese characters could have been working with no further ado, but the default font by Sphinx for `xelatex` is GNU FreeFont (FreeSerif, FreeSans, FreeMono).  If you replace that with some font which will have the glyphs needed inclusive of those for Chinese, then the above is not needed. GNU FreeFont is GNU, and it covers Arabic, Cyrillic, Greek, Hebrew and Latin scripts (among others) but not Chinese ideograms...

The problem for this PR is that don't know what are the constraints for this project regarding fonts to use for PDF builds.  I take here "conservative" approach to give a configuration allowing build on any system having a LaTeX installation based upon TeXLive, such as Debian based distros, and not using System Fonts (although XeLaTeX is perfectly able to use System Fonts).  Myself, I had only a partial TeXLive install and I issued on my macos "tlmgr install fandol" to add the Fandol fonts to my system, and on Ubuntu it would go via using `apt` to install `texlive-lang-chinese`.  But one can also use system fonts.

Reply:
Ah well... feels like there should be a nicer solution, but this seems OK to me especially for now.  Not like we add new Chinese characters often.

Reply:
@seberg I do wish there was some way to simply tell `xelatex` once and for all "if you hit against a Chinese character, use FandolSong-Regular.otf to render it".  Alas, I am not aware of a general method to achieve that, i.e. to handle multilingual documents without extra mark-up like `\selectlanguage{chinese}`.  This is indeed very cumbersome as it means monitoring LaTeX builds for messages in its logs about missing characters and manually updating the list of problematic ones: currently only Chinese characters from release notes, with author names in lists of contributions.